Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -190,6 +193,11 @@ public async Task ValidateContestPassword(int id, bool official, string password
                     .FirstOrDefault();
             });
 
+            participationModel.ActiveParticipantsCount = this.contestParticipantsCacheService.GetParticipantsCountByContestAndIsOfficial(
+                    model.ContestId,
+                    true);
+            participationModel.TotalParticipantsCount = this.contestParticipantsCacheService.GetParticipantsCountByContest(model.ContestId);
```

Comment:
We should display `Total` for the chosen contest mode - Total for Compete or Total for Practice, never Total combined.